engines/asylum/puzzles/boardsalvation.cpp

```diff
@@ -131,7 +131,32 @@ static const PuzzleBoard::PuzzleData puzzleSalvationData[] = {
 		false,
 		12, 0,
 		"\xE4 \xF2 \xE5 \xF9 \xE9 \xE4   \xEC \xE0 "
-	}
+	},
+	// Basque
+	{
+		31,
+		kGameFlag281,
+		431,
```

Comment:
You need to increase this value (455 would work), otherwise the last letter won't be drawn